backend/repository/repository.go

```diff
@@ -6,25 +6,24 @@ import (
 	"delta.nitt.edu/dion/config"
 	"delta.nitt.edu/dion/models"
 	"go.uber.org/fx"
-	"go.uber.org/zap"
 	"gorm.io/driver/postgres"
 	"gorm.io/gorm"
 )
 
 type Params struct {
 	fx.In
-	Conf   *config.Config
-	Logger *zap.Logger
+	// Conf   *config.Config
+	// Logger *zap.Logger
 }
 
 func New(p Params) (*gorm.DB, error) {
```

Comment:
We can probably remove the New function and actually just use [init](https://www.digitalocean.com/community/tutorials/understanding-init-in-go) function. I think it'll be more cleaner this way. DI not worth it in this project 😔